waypoint_x:
  - 1.0
  - 5.0
  - 5.0
  - 3.0
  - 1.0
waypoint_y:
  - 1.0
  - 1.0
  - 5.0
  - 7.0
  - 5.0
trans_vel: 0.5 #The translational velocity of the robot
rot_vel: 0.5 #The rotational velocity of the robot
frequency: 60 # The frequency of the control loop
linear_threshold: 0.01 
angular_threshold: 0.01